lib/order.rb

```diff
@@ -0,0 +1,49 @@
+require_relative "send_sms"
+require 'dotenv/load'
+
+class Order
+  def initialize(dish, quantity = 1)
+    @selected_dishes = []
+    quantity.times { @selected_dishes.push(dish) }
+    @confirmed = false
+    view_total
+  end
+
+  def update(dish, quantity = 1)
+    quantity.times { @selected_dishes.push(dish) }
+    view_total
+  end
+
+  def view_selected
+    @selected_dishes
+  end
+
+  def view_total
+    total = 0
+    @selected_dishes.each do |dish|
+      total += dish.price
+    end
+    @total = total
+  end
+
+  def confirm_order(
+    _to = ENV['TWILIO_TO_NUMBER'],
+    message_class = Sms,
+    time = calculate_time
+  )
+    sms = message_class.new
```

Comment:
I like how you've made the type of message able to change, but your variable name suggests SMS. Could the variable name change to something more generic?